msp/mspimpl.go

```diff
@@ -890,9 +890,15 @@ func (msp *bccspmsp) getCertificationChainIdentifierFromChain(chain []*x509.Cert
 // is regenerated to have a Low-S signature.
 func (msp *bccspmsp) sanitizeCert(cert *x509.Certificate) (*x509.Certificate, error) {
 	if isECDSASignedCert(cert) {
+		validityOpts := msp.getValidityOptsForCert(cert)
+		if bytes.Equal(cert.RawSubject, cert.RawIssuer) && cert.IsCA {
+			validityOpts.Roots = nil
```

Comment:
why do you set these `validityOpts` if you return right after in line 897?

Reply:
It's a leftover, I will remove it.